src/LibraryViewExtension/web/library/layoutSpecs.json

```diff
@@ -184,15 +184,6 @@
               "iconUrl": "",
               "elementType": "group",
               "include": [
-                {
-                  "path": "BuiltIn.List.ContainsKey"
```

Comment:
can you also update librariejs with these same changes?

Reply:
Sorry what does this comment mean?

Reply:
Ahh got it :-)

Reply:
@mjkkirschner There seem to be a lot of deltas between this file and the one found in the lib repo.  I will follow up with @ramramps to see if the changes need to be sync'd

Reply:
@saintentropy Not necessary. the clients can have its Layoutspec.json.  I don't think we should have a parity between the one in Librarie.js and Dynamo.